src/webprojects/fundraising.md

```diff
@@ -1,19 +1,17 @@
-# Fundraising website: how to localize and test
+# Mozilla Donate website: how to localize and test
 
-[donate.mozilla.org](https://donate.mozilla.org) is our platform to collect donations. The main form is for Mozilla Foundation, but it also hosts a separate form for donations to Thunderbird.
-
-This website relies on third parties to handle payment. The localization of the payment system for debit/credit cards is managed directly by Stripe and it’s not possible to add or change translations.
+[donate.mozilla.org](https://donate.mozilla.org) is our platform to collect donations. The main instance is for donations to the Mozilla Foundation, but there is also a separate instance for donations to Thunderbird.
 
 ## Testing
 
 Submitted translations are automatically deployed to the staging server, so you can test them a few minutes after they have been translated.
 
-* Staging server: https://donate.mofostaging.net/​.
+* Staging server: https://donate-wagtail.mofostaging.net/​.
 * Production server: https://donate.mozilla.org/.
 
 ### Testing the payment flow
 
-You may want to test the complete user journey on the website, but you don’t have to make an actual donation for that. Stripe has a [test mode](https://stripe.com/docs/testing#cards) that is enabled on the staging server only. Simply use the payment info from this page on the staging server and you should be able to simulate a payment. For instance, you can select an amount of money, click on donate by credit/debit card then fill out the Stripe form with the testing info. Make sure errors are properly localized, as well as the page displayed when the donation has been successful. You should also receive a confirmation email in your language.
+You may want to test the complete user journey on the website, but you don’t have to make an actual donation for that. Braintree (our payment processor for debit/credit cards) has a [test mode](https://developers.braintreepayments.com/guides/credit-cards/testing-go-live/python#valid-card-numbers) that is enabled on the staging server only. Simply use the payment info from this page on the staging server and you should be able to simulate a payment. For instance, you can select an amount of money, click on donate by credit/debit card then fill out the form with the testing info. Make sure errors are properly localized, as well as the page displayed when the donation has been successful.
```

Comment:
The last sentence seems a bit off because of the order?

```
Make sure errors, as well as the page displayed when the donation has been successful, are properly localized.
```